### PR TITLE
Switch to using defusedxml for parsing xml

### PR DESCRIPTION
**Pull request by kwarzinski**
This change fixes a vulnerability xml bomb attacks

### CODE REVIEW

File: simplexmlapi/node.py [hunk 1]
```diff
@@ -1,5 +1,6 @@
 from xml.dom.minicompat import *
-from xml.dom.minidom import Node, parseString, Attr
```
Comment (advocatejake) [inline simplexmlapi/node.py:2]: Attr import removed, able to comment it appears after logging in

**Review by advocatejake — APPROVED** [review]
looks clean to me.

**Review by fsant02 — APPROVED** [review]

**Review by erin-doyle — APPROVED** [review]

File: setup.py [hunk 2]
```diff
@@ -31,5 +31,5 @@
       packages=find_packages(exclude=['ez_setup', 'examples', 'tests']),
       platforms=['any'],
       zip_safe=False,
-      install_requires=['six'],
+      install_requires=['six', 'defusedxml'],
```
Comment (matteius) [inline setup.py]: I realize this is a fork of a third party library and there is no requirements.txt but my preference is not to use the install_requires.  I think we can do so here given the third party library thing with no alt-way of specifying dependencies, but make sure anywhere use this in a project that we actually call out an pin the version of difusedxml in the requirements.txt

Comment (mpistrang) [reply setup.py]: Is there a reason why defusedxml is not pinned a version in the setup.py, too?   Will simplexmlapi really work with any future or past version of defusedxml?

Comment (erin-doyle) [reply setup.py]: Yeah I think those are good points about pinning.

Comment (kwarzinski) [reply setup.py]: How about I pin it to >=0.6.0 as that seems to be the latest version?

**Review by ethurgood — APPROVED** [review]
nice!

**Pull request merged**